### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,5 +9,22 @@
...
+                 'License :: OSI Approved :: MIT License',
+
+                 'Operating System :: OS Independent',
+
+                 'Programming Language :: Python :: Implementation :: CPython',
+                 'Programming Language :: Python :: 3.3',
+                 'Programming Language :: Python :: 3.4',
+                 'Programming Language :: Python :: 3.5',
+                 'Programming Language :: Python :: 3 :: Only'],
 )
```
Comment (AbdealiLoKo) [inline setup.py:31]: Wrong indentation ?
What sort of code convention do you wish to adopt in this repo ? (If it's same as coala, then the round bracket should be on the line above)

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from distutils.core import setup
 setup(
-  name = 'coala-decorators',
-  packages = ['coala-decorators'],
+  name = 'coala_decorators',
```
Comment (AbdealiLoKo) [inline setup.py:5]: Note that the name can still have hyphen.
For example, the coala-bears repo has `name = "coala-bears"` and `packages = ["bears"]`
Again, This is your pereference, so I'm going to ack it anyway

File: setup.py [hunk 3]
```diff
@@ -1,8 +1,10 @@
+from coala_decorators import VERSION
 from distutils.core import setup
+
 setup(
   name = 'coala_decorators',
   packages = ['coala_decorators'],
-  version = '0.1.1',
+  version = VERSION,
```
Comment (AbdealiLoKo) [inline setup.py:7]: Um, I'm fairly sure that this shouldnt work.
You cant import a non-python file like `from <pkg name> import FILENAME`. (I hope Im not wrong here)
In the `coala` code, theres a small line in `coalib/__init__.py` which reads the VERSION file and saves it in the VERSION and **version** variables.
Check https://github.com/coala-analyzer/coala/blob/master/coalib/__init__.py#L5

Comment (Adrianzatreanu) [inline setup.py:7]: i think this would ought to work now?

**Comment by sils** [comment]
ack https://github.com/coala-analyzer/coala-decorators/pull/12/commits/a3b16f577fd18bea6c86ee6225512321da13b36b

**Comment by sils** [comment]
ack https://github.com/coala-analyzer/coala-decorators/pull/12/commits/61e87a7622040354b8ad41182f94fef7a6826fcb https://github.com/coala-analyzer/coala-decorators/pull/12/commits/e6c7031e3ee9cce823dc5fbccfb6fcbbe2d355cc https://github.com/coala-analyzer/coala-decorators/pull/12/commits/439c2d5e1eb6fcfc9cf0f7edfb6809126c1d9401 https://github.com/coala-analyzer/coala-decorators/pull/12/commits/5917cf68865127edb5aba716384eb50e80069f04

**Comment by sils** [comment]
@rultor merge

**Pull request merged**